controllers/storagerequest/storagerequest_controller.go

```diff
@@ -243,7 +243,7 @@ func (r *StorageRequestReconciler) initPhase() error {
 		} else {
 			return fmt.Errorf("invalid number of CephBlockPoolRadosNamespaces for storage consumer %q: found %d, expecting 0 or 1", r.storageConsumer.Name, rnsItemsLen)
 		}
-	} else if r.StorageRequest.Spec.Type == "sharedfilesystem" {
+	} else if strings.ToLower(r.StorageRequest.Spec.Type) == "sharedfile" {
```

Comment:
For completeness I will add this suggestion to the last one 

```suggestion
	} else if isSharedFileRequest(r.StorageRequest.Spec.Type) {
```

Reply:
done.